tests/components/fyta/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,39 @@
+# serializer version: 1
+# name: test_entry_diagnostics
+  dict({
+    'config_entry': dict({
+      'data': dict({
+        'access_token': '123xyz',
+        'expiration': '2030-12-31T10:00:00+00:00',
+        'password': '**REDACTED**',
+        'username': '**REDACTED**',
+      }),
+      'disabled_by': None,
+      'domain': 'fyta',
+      'entry_id': 'ce5f5431554d101905d31797e1232da8',
+      'minor_version': 2,
+      'options': dict({
+      }),
+      'pref_disable_new_entities': False,
+      'pref_disable_polling': False,
+      'source': 'user',
+      'title': 'fyta_user',
+      'unique_id': None,
```

Comment:
Please remind me, did we set the unique id in the config flow?

Reply:
Not explicitly. But isn't the unique id set with `async_create_entry`?